pinot-plugins/pinot-input-format/pinot-orc/pom.xml

```diff
@@ -70,5 +70,9 @@
       <artifactId>xml-apis</artifactId>
       <scope>compile</scope>
     </dependency>
+    <dependency>
+      <groupId>com.google.protobuf</groupId>
+      <artifactId>protobuf-java</artifactId>
```

Comment:
We should set this in the dependency management section of the root pom. That way it will be enforced for any transitive dependency

Reply:
It is defined in root pom. However the shading packages another version, so need to explicitly set it here.

Reply:
I don't understand why that is happening, but lets merge it as it is suggested here.